code/modules/mob/living/carbon/human/species.dm

```diff
@@ -215,6 +215,10 @@
 	var/dynamic_hair_suffix = "" //if this is non-null, and hair+suffix matches an iconstate, then we render that hair instead
 	var/dynamic_fhair_suffix = ""
 
+	//for augmented heads
+	if(HD.status == BODYPART_ROBOTIC)
```

Comment:
No, this is intentional.
For your needs, make a new var ```draws_hair``` and set it ```TRUE``` by default, and ```FALSE``` for yours, then make the check:

```
if(!HD.draws_hair)
    return 
```